google/cloud/storage/internal/list_object_acl_request_test.cc

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/list_object_acl_request.h"
+#include <gmock/gmock.h>
+
+namespace google {
+namespace cloud {
+namespace storage {
+inline namespace STORAGE_CLIENT_NS {
+namespace internal {
+namespace {
+
+TEST(ListObjectAclRequestTest, Simple) {
+  ListObjectAclRequest request("my-bucket", "my-object");
+  EXPECT_EQ("my-bucket", request.bucket_name());
+  EXPECT_EQ("my-object", request.object_name());
+}
+
+using ::testing::HasSubstr;
+
+TEST(ListObjectAclRequestTest, Stream) {
+  ListObjectAclRequest request("my-bucket", "my-object");
+  request.set_multiple_parameters(UserProject("my-project"), Generation(7));
+  std::ostringstream os;
+  os << request;
+  EXPECT_THAT(os.str(), HasSubstr("userProject=my-project"));
```

Comment:
Rather than calling os.str() repeatedly, could you call it once and store the string in a variable?

Reply:
Done.